### PR TITLE
work in progress -- introduction to snakemake for automation

### PR DESCRIPTION
**Pull request by taylorreiter**
starts snakemake for automation. 

PR author:
- [ ] pull request is ready for review & merge

Things to check:

- [x] tutorial resets working directory at beginning with a `cd ~/``
- [ ] tutorial contains relevant `conda install -y` commands at the beginning
- [ ] tutorial has learning objectives at top

### CODE REVIEW

**Review by SichongP — COMMENTED** [review]
It's not clear to me which directory they should be working in during this session. Can we make dir organization clearer?

**Comment by taylorreiter** [comment]
Thanks @sichongP, that's a really good point! I totally forgot that that would not be intuitive for everyone :)

File: snakemake_for_automation.md [hunk 1]
```diff
@@ -0,0 +1,354 @@
...
+        "fastqc_raw/ERR458493_fastqc.zip"
+    shell:'''
+    fastqc -o fastqc_raw {input}
+    '''
+
+rule unzip_fastqc_raw:
+    input: "fastqc_raw/ERR458493_fastqc.zip"
+    output: "fastqc_raw/ERR458493_fastqc/summary.txt"
+    shell:'''
+    unzip -d fastqc_raw/ {input}
```
Comment (SichongP) [inline snakemake_for_automation.md:286]:
```suggestion
    unzip -d fastqc_raw/ {input}
    '''
```

Comment (taylorreiter) [reply snakemake_for_automation.md:286]: @SichongP I don't see how your suggestion is any different from what is currently there?

Comment (SichongP) [reply snakemake_for_automation.md:286]: The `'''` at the end of shell command is missing. It gave a syntax error when I ran it.

Comment (taylorreiter) [reply snakemake_for_automation.md:286]: Ah! I didn't see it was missing, I was only looking at line 286. Good catch, thank you :)

**Pull request merged**